manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml

```diff
@@ -16,24 +16,4 @@ spec:
       prune: true
       selfHeal: true
     syncOptions:
-    - Validate=false
```

Comment:
did you also mean to remove this flag?

Reply:
I don't think it's removing it.. it's the indentation.. I'll remove the change.